clang/lib/Tooling/DependencyScanning/ModuleDepCollector.cpp

```diff
@@ -110,7 +110,7 @@ void ModuleDepCollector::addOutputPaths(CowCompilerInvocation &CI,
 }
 
 static CowCompilerInvocation
-makeCommonInvocationForModuleBuild(CompilerInvocation CI) {
+makeCommonInvocationForModuleBuild(const CompilerInvocation &CI) {
```

Comment:
This does not compile, `CI` is mutated in this function.

Reply:
>>This does not compile, CI is mutated in this function.
Yes, i missed that. Thank you @Xazax-hun for reviews. 

I have updated patch.